pkg/fakedata/generator.go

```diff
@@ -177,6 +178,27 @@ var enum = func(column Column) string {
 	return withEnum(enum)(column)
 }
 
+var fileCache map[string][]string
```

Comment:
nice! As the program works right now I think this is a cool approach.

It's interesting to notice that this is needed because the design of the program is still suboptimal. Right now, all the Constraints are computed for every single row which makes no sense from a perf perspective :D. I've been wanting to start working on it for a while (but only because it's fun!) but I had no real reason. Now that I see this I feel more compelled to start working on that too.

It was only a digression though (kind of a way for me to start thinking about this more concretely). I think what you did here is pretty good for the current design